content/helpers.go

```diff
@@ -185,14 +181,6 @@ func Copy(ctx context.Context, cw Writer, or io.Reader, size int64, expected dig
 		}
 		if err := cw.Commit(ctx, size, expected, opts...); err != nil {
 			if errors.Is(err, ErrReset) {
-				ws, err := cw.Status()
-				if err != nil {
-					return fmt.Errorf("failed to get status: %w", err)
-				}
-				r, err = seekReader(or, ws.Offset, size)
-				if err != nil {
```

Comment:
Q: Is this removed because we are not expecting to resume once we have reached commit stage?

Reply:
The change shouldn't change the previous behavior (at least that's my intent :)).

The 2nd and 3rd `cw.Status + seekReader` calls I removed are just merged into the 1st one, now at the beginning of the for loop (notice `continue` are still there, so they will just go to the next loop (with `i > 0`) and do `cw.Status + seekReader` there. Previously, they just do `cw.Status + seekReader` first and then `continue`.